packages/runner/README.md

```diff
@@ -8,56 +8,106 @@ Run scanning for vulnerabilities just from your unit tests on CI phase.
 
-2.  Setup `SecRunner`:
+Once it is done, create a configuration object. Single required option is NeuraLegion `cluster` domain you are going to use, e.g. `app.neuralegion.com` as the main one:
 
-    - Create `SecRunner` instance, providing either `ConfigurationOptions` or `Confiruration` instance as constructor argument.
-      Single required option is NeuraLegion `cluster` domain you are going to use, e.g. `app.neuralegion.com` as the main one.
-      See other available advanced config options [here](https://github.com/NeuraLegion/sec-tester-js/tree/master/packages/core#options).
+```ts
+import { Confiruration } from '@secbox/core';
+
+const configuration = new Confiruration({ cluster: 'app.neuralegion.com' });
+```
+
+### Setup runner
+
+To set up a runner, create `SecRunner` instance passing a previously created configuration as follows:
+
+```ts
+import { Confiruration } from '@secbox/core';
+import { SecRunner } from '@secbox/runner';
+
+const configuration = new Confiruration({ cluster: 'app.neuralegion.com' });
+const runner = new SecRunner(configuration);
+
+// or
+
+const runner2 = new SecRunner({ cluster: 'app.neuralegion.com' });
+```
+
+After that, you have to initialize a `SecRunner` instance:
+
+```ts
+await runner.init();
+```
+
+The runner is now ready to perform your tests, but you have to create a scan.
 
-    - Initialize `SecRunner` instance by calling `init()`.
+To dispose a runner, you just need to call the `clear` method:
 
-3.  Setup `SecScan`:
+```ts
+await runner.clear();
+```
+
+### Starting scan
+
+To start scanning your application, first you have to create a `SecScan` instance, as shown below:
+
+```ts
+const scan = runner.createScan({ tests: [TestType.XSS] });
+```
+
+Below you will find a list of parameters that can be used to configure a `Scan`:
 
-    - Create `SecScan` instance by calling `createScan(opts: SecScanOptions)` of `SecRunner` instance providing at least list of `tests`
+| Option                 | Description                                                                                                                                                                                   |
+| ---------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `tests`                | The list of tests to be performed against the target application. [Learn more about tests](https://docs.brightsec.com/docs/vulnerability-guide)                                               |
+| `smart`                | Determine whether scan is smart or simple. Use automatic smart decisions such as: parameter skipping, detection phases, etc. to minimize scan time. Enabled by default.                       |
```

Comment:
i like more wording in prev version. "Determine whether scan is smart or simple." imo can't be clear for user

Reply:
It makes sense to adjust the wording in the same table in the scan package too. I'll update